toolchain/check/decl_name_stack.cpp

```diff
@@ -188,25 +188,24 @@ auto DeclNameStack::TryResolveQualifier(NameContext& name_context,
       if (auto class_decl = context_->insts()
                                 .Get(name_context.resolved_inst_id)
                                 .TryAs<SemIR::ClassDecl>()) {
-        CARBON_DIAGNOSTIC(QualifiedDeclInIncompleteClassScope, Error,
+        CARBON_DIAGNOSTIC(QualifiedNameInIncompleteClassScope, Error,
```

Comment:
I think this is also talking about a erroneous declaration due to qualification.

Reply:
Done.